docs/pages/bare/error-recovery.md

```diff
@@ -0,0 +1,67 @@
+
+> Disclaimer: the behavior documented below is subject to change and should not be relied upon. Always test your code carefully and thoroughly in production-like environments before publishing updates.
+
+> The behavior documented on this page applies to bare apps as well as apps built with EAS Build, **not** apps built with `expo build:ios` or `expo build:android` (which have a separate error recovery mechanism that uses [`expo-error-recovery`](/versions/latest/sdk/error-recovery)).
+
+## Help! I published a broken update to production. What should I do?
+
+First of all, don't panic. Mistakes happen; most likely, everything will be fine.
+
+The important thing is to **publish a new update with a fix as soon as possible (though not before you are 100% confident in your fix).** The error recovery mechanism in `expo-updates` will ensure that in most cases, even users who have already downloaded the broken update should be able to get the fix.
+
+The first thing to try is rolling back to an older update that you know was working. **However, this may not always be safe;** your broken update may, for example, have modified persistent state (such as data stored in AsyncStorage or on the device's file system) in a non-backwards-compatible way. It's important to test in a staging environment that emulates an end user's device state as closely as possible—i.e. load the broken update and then roll back.
+
+If you can identify an older update that is safe to roll back to, you can do so using [EAS Update's `republish` option](/eas-update/eas-update-and-eas-cli/#republish-a-previous-update-within-a-branch). If you're using classic Expo-hosted updates (`expo publish`) or hosting your updates on your own server, you'll need to publish a new update with your old code in order to generate an update with a new ID and publish date.
+
+If you cannot identify an older update that is safe to roll back to, you'll need to fix forward. While it's best to roll out a fix as quickly as possible, you should take the time to ensure your fix is solid, and know that even users who download your broken update in the meantime should be able to download your fix.
+
+If you'd like more details about how this works, read on.
+
+## Explaining the error recovery flow
+
+The error recovery flow is intended to be as lightweight as possible. It is not a full safety net that protects your end users from the results of errors; in many cases users will still see a crash.
+
+Rather, the purpose is to prevent updates from "bricking" your app (causing a crash on launch before the app can check for updates, making the app unusuable until uninstalled and reinstalled) by ensuring that in as many cases as possible, the app has the opportunity to download a new update and fix itself.
```

Comment:
```suggestion
Rather, the purpose is to prevent updates from "bricking" your app (causing a crash on launch before the app can check for updates, making the app unusable until uninstalled and reinstalled) by ensuring that in as many cases as possible, the app has the opportunity to download a new update and fix itself.
```